PlayerPanel.js

```diff
@@ -34,87 +34,121 @@ function update_pclist() {
 
 
 	});
-
-
-
+	const addPartyButtonContainer = $("<div class='add-party-container'></div>");
+	const addPartyButton = $("<button id='add-party'>ADD PARTY</button>");
+	addPartyButton.on('click', () => {
+		window.pcs.forEach(function (p, i) {

```

Comment:
NIT

May be worth givng `p` a better var name to reduce cognitive load. I.E. Is this player, party? Don't make us think.